frontend/packages/schema-editor/src/components/SchemaTree/SchemaNode/ActionButtons/ActionButtons.tsx

```diff
@@ -8,6 +8,7 @@ import { ActionButton } from './ActionButton';
 import { AddPropertyMenu } from './AddPropertyMenu';
 import { useSavableSchemaModel } from '../../../../hooks/useSavableSchemaModel';
 import { SavableSchemaModel } from '../../../../classes/SavableSchemaModel';
+import { useSchemaEditorAppContext } from '@altinn/schema-editor/hooks/useSchemaEditorAppContext';
```

Comment:
I think we should use absolute path imports here as it is from within the same package.

Reply:
You're right. I hate my code editor.

Reply:
It's the same with mine, so annoying